01-js-intro-and-jquery/app.js

```diff
@@ -0,0 +1,5 @@
+var main = function() {
+    $('.hidden').fadeIn(5000);
+    $( '.alias' ).focus().delay(5000); /*can't set the delay*/
+};
```

Comment:
You must leave a white space between variables definitions and the source "logic" or the next source block.